appj.adoc

```diff
@@ -25,52 +25,105 @@ When an interpolation method is referred to as linear or quadratic, it means tha
+| Conversion from cartesian vector `(x, y, z)` to geocentric `(latitude, longitude)`| `(lat, lon) = fxyz2ll(v) = (atan2(v.y, v.x), atan2(z, sqrt(v.x * v.x + v.y * v.y))` +
+ 
+| Conversion from `(azimuth, zenith)` angles to cartesian vector `(x, y, z)` | `(x, y, z) = faz2xyz(az) = (sin⁡(az.zenith) * sin⁡(az.azimuth), sin⁡(az.zenith) * cos⁡(az.azimuth), cos⁡(az.zenith))` +
+| Conversion from cartesian vector `(x, y, z)` to `(azimuth, zenith)` angles | `(azimuth, zenith) = fxyz2az(v) = (atan2(y, x), atan2(sqrt(x * x + y * y), z)` +
+
+| Calculate the interpolation coefficient `c` for quadratic interpolation of one coordinate in one dimension between tie points `a` and `b` from given coordinate value `u` at `s` | `c = fc1(ua, ub, u, s) = ((u - (1 - s)*ua - s*ub)/((1 - s)*s)` + 
+| Calculate the interpolation coefficients `(c.1, c.2)` for quadratic interpolation of two coordinates in one dimension between tie points `a` and `b` from given coordinate values `(u.1, u.2)` at `s` | `(c.1, c.2) = fc2(ua, ub, u, s) = (fc1(ua.1, ub.1, u.1, s), fc1(ua.2, ub.2, u.2, s))` + 
+| Calculate the interpolation coefficients `(c.1, c.2, c.3)` for quadratic interpolation of three coordinates in one dimension between tie points `a` and `b` from given coordinate values `(u.1, u.2, u.3)` at `s` | `(c.1, c.2, c.3) = fc3(ua, ub, u, s) = (fc1(ua.1, ub.1, u.1, s), fc1(ua.2, ub.2, u.2, s), fc1(ua.3, ub.3, u.3, s))` + 
+
+| Calculate the linear interpolation value of one coordinate `u` in one dimension between tie points `a` and `b`   | `u = fl1(ua, ub, s) = (1 - s)*ua + s*u2`; + 
+
+| Calculate the quadratic interpolation value of one coordinate `u` in one dimension between tie points `a` and `b` with interpolation coefficient `c`   | `u = fq1(ua, ub, c, s) = (1 - s)*ua + (1 - s)*s1*c + s*u2`; + 
+
+| Calculate the quadratic interpolation values of two coordinates `(u.1, u.2)` in one dimension between tie points `a` and `b` with interpolation coefficients `(c.1, c.2)`  | `(u.1, u.2) = fq2(ua, ub, c, s) = (fq1(ua.1, ua.2, c.1, s), fq1(ua.1, ua.2, c.1, s))``; +
+
+| Calculate the quadratic interpolation values of three coordinates `(u.1, u.2, u.3)` in one dimension between tie points `a` and `b` with interpolation coefficients `(c.1, c.2, c.3)`  | `(u.1, u.2, u.3) = fq3(ua, ub, c, s) = (fq1(ua.1, ub.1, c.1, s), fq1(ua.2, ub.2, c.2, s), fq1(ua.3 ub.3, c.3, s))`; +
+
+| Vector Sum | `(x, y, z) = va + vb = (va.x + vb.x, va.y + vb.y, va.z + vb.z)` + 
+| Vector Difference | `(x, y, z) = va - vb = (va.x - vb.x, va.y - vb.y, va.z - vb.z)` + 
+ 
+| Vector Cross Product | `(x, y, z) = fcross(va, vb) = (va.y*vb.z - va.z*vb.y, va.z*vb.x - vva.zx*b.z, va.x*vb.y - va.y*vb.x)` + 
+
+| Normalised Vector | `(x, y, z) = norm(v) = (v.x, v.y, v.z) / sqrt (v.x*v.x + v.y*v.y + v.z*v.z)` + 
+
+| Vector Dot Product | `d = fdot(va, vb) = va.x*vb.x + va.y*vb.y, va.z*vb.z`
+
+| Vector from ce, ca  | `(x, y, z) = fp(va, vb, ce, ca) = norm(0.5*fplus(va, vb) + ce*fminus(vb, va) + ca*fcross(vb, va))`
```

Comment:
Adding a defintion/description for ce, ca might be necessary

Reply:
Yes, that is needed. Let's discuss where to put it.